server/repository/src/db_diesel/purchase_order_line_row.rs

```diff
@@ -16,8 +15,9 @@ table! {
     purchase_order_line (id) {
         id ->  Text,
         purchase_order_id -> Text,
-        line_number -> Integer,
+        line_number -> Nullable<BigInt>,
         item_link_id -> Nullable<Text>,
+        item_name ->  Nullable<Text>,
```

Comment:
item name was already in the migration - but had been missed in previous PR from OMS purchase order line row type